test/samples/local/foundry_sign_addr/contracts/test/Cheatcodes.t.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "../src/Counter.sol";
+
+contract CheatcodeTest is Test {
+    function setUp() public {
+    }
+
+    function testSign() public {
+	    bytes32 digest = "test";
+	    uint256 privKey = 0x6b911fd37cdf5c81d4c0adb1ab7fa822ed253ab0ad9aa18d77257c88b29b718e;
+	    // Correct address
+	    address addr = 0x16bB6031CBF3a12B899aB99D96B64b7bbD719705;
+	    // Incorrect address
+	    address badAddr = 0x16bb6031CbF3a12B899AB99d96b64b7BBD719706;
+
+	    (uint8 v, bytes32 r, bytes32 s) = vm.sign(privKey, digest);
+
+	    address rec = ecrecover(digest, v, r, s);
+
+	    assertEq(rec, badAddr);
```

Comment:
Variable says `badAddr`. Why do we need `addr` then? What do we actually expect here?

Reply:
Just figured that we intentionally want test to fail. In that case - things are fine. Still, I would be grateful if ther would be a comment, that explicitly state our intentions (describe the case in words, if possible).

Reply:
Sorry, I have to re-do the whole campaign for the one comment, so for now will skip. Hopefully we won't have to look at this code again